pkg/osvscanner/vulnerability_result.go

```diff
@@ -79,7 +79,8 @@ func buildVulnerabilityResults(
 
 		// For Debian-based ecosystems, mark unimportant vulnerabilities within the package.
 		// Debian ecosystems may be listed with a version number, such as "Debian:10".
-		if strings.HasPrefix(pkg.Package.Ecosystem, string(models.EcosystemDebian)) {
+		if strings.HasPrefix(pkg.Package.Ecosystem, string(models.EcosystemDebian)) ||
```

Comment:
Can we move the check into the isUnimportant function? This way the setUnimportant function is not tied to a specific ecosystem.

Reply:
I think it's better to keep the check here. This check determines whether the scanned package itself belongs to Ubuntu or Debian, while `isUnimportant` checks if a vulnerability has Ubuntu/Debian related information and is marked as unimportant. 

Once we have an ecosystem severity field (I saw some discussion on osv-schema about this), I can update code here to make it less hardcoded.